web/src/frontend/src/nav-soknad/containers/StegMedNavigasjon.tsx

```diff
@@ -96,6 +97,13 @@ class StegMedNavigasjon extends React.Component<Props, {}> {
 	handleGaVidere(aktivtSteg: SkjemaSteg, brukerBehandlingId: string) {
 		if (aktivtSteg.type === SkjemaStegType.oppsummering) {
 			if (this.props.oppsummeringBekreftet) {
+
+				const typeAdresseFaktum: Faktum = finnFaktum("kontakt.system.oppholdsadresse.valg", this.props.fakta);
```

Comment:
En idè kunne være å erstatte få linjene med kode med et funksjonskall som har et forklarende navn. For eksempel `this.props.dispatch(loggAdresseTypeTilGrafana())`. Det fungerer ofte mye bedre enn kommentarer, som gjerne ikke blir vedlikeholdt.